chains/evm/calls/transactor/transact.go

```diff
@@ -0,0 +1,96 @@
+package transactor
+
+import (
+	"context"
+	"fmt"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/client"
+	"github.com/imdario/mergo"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/rs/zerolog/log"
+)
+
+type TransactOptions struct {
+	GasLimit uint64
+	GasPrice *big.Int
+	Value    *big.Int
+}
+
+func DEFAULT_TRANSACTION_OPTIONS() TransactOptions {
```

Comment:
If you wanna name it like that it can be a variable instead of a function.
Making it a function with snake case doesn't really make sense.
Also, if it doesn't do anything might as well be a variable.